pkg/signup/service/signup_service.go

```diff
@@ -85,8 +87,7 @@ func (s *ServiceImpl) newUserSignup(ctx *gin.Context) (*toolchainv1alpha1.UserSi
 	for _, bu := range bannedUsers.Items {
 		// If the user has been banned, return an error
 		if bu.Spec.Email == userEmail {
-			return nil, apierrors.NewForbidden(schema.GroupResource{}, "",
-				errs.New("Access to the Developer Sandbox has been suspended due to suspicious activity or detected abuse."))
+			return nil, apierrors.NewForbidden(schema.GroupResource{}, "", errs.New(BannedUserMessage))
```

Comment:
nitpick, but `apierrors.NewForbidden(schema.GroupResource{}, "", errs.New(BannedUserMessage))` can be made into a package-level var and be just returned from here instead of recreating it each time.

That said, the effect will be negligible.

Reply:
done in ac0a1b7